run_ibis_benchmark.py

```diff
@@ -402,10 +405,14 @@ def main():
                         )
 
                 for result_etl in etl_results:
+                    if ignore_fields_for_bd_report_etl:
+                        remove_fields_from_dict(result_etl, ignore_fields_for_bd_report_etl)
```

Comment:
```suggestion
                    remove_fields_from_dict(result_etl, ignore_fields_for_bd_report_etl)
```
as you've fixed `remove_fields_from_dict()` you no longer need to do that `if`